### ISSUE TITLE
File ** returns a glob, but Glob ** returns PathFinder

### ISSUE THREAD

**Issue by eed3si9n** [sbt/sbt#4648]
## steps

Try loading play-ws's build - https://github.com/playframework/play-ws

## problem

```
[error] /xxx/play-ws/project/CleanShadedPlugin.scala:46:10: value get is not a member of Object
[error] possible cause: maybe a semicolon is missing before `value get'?
[error]       }).get
```

## expectation

It loads, keeping source compat with 1.2.x.

`base ** "*"` returns a Glob now, but if you chain `**` method it goes back to PathFinder, so you get a mix of globs and PathFinders now.

## notes

```scala
    def query(base: File, param: Option[ModuleParam], org: String, name: String): Seq[File] =
      (param match {
        case None                                   => base ** ("*" + org + "*") ** ("*" + name + "*")
        case Some(ModuleParam("*", None))           => base ** "*"
        case Some(ModuleParam(o, None | Some("*"))) => base ** ("*" + o + "*") ** "*"
        case Some(ModuleParam(o, Some(n)))          => base ** ("*" + o + "*") ** ("*" + n + "*")
      }).get
```

**Comment by dwijnand** [comment]
One workaround is to explicitly use the older, now lower priority, File to PathFinder conversion, via `PathFinder.apply`.  For example https://github.com/playframework/play-ws/pull/338/commits/6a9d4bc927de89a063523cc8cc9962fe23a9f694 (part of https://github.com/playframework/play-ws/pull/338).

**Comment by eed3si9n** [comment]
Fixed in #4664 
Confirmed with M4.
